File: Multi_Language_CodeExamples/dart/calculator.dart
class Calculator {
  int add(int a, int b) => a + b;
  int multiply(int a, int b) => a * b;
}

void main() {
  var calc = Calculator();
  print('5 + 3 = ${calc.add(5, 3)}');
  print('5 * 3 = ${calc.multiply(5, 3)}');
}

File: Multi_Language_CodeExamples/kotlin/calculator.kt
class Calculator {
    fun add(a: Int, b: Int): Int = a + b
    fun multiply(a: Int, b: Int): Int = a * b
}

fun main() {
    val calc = Calculator()
    println("5 + 3 = ${calc.add(5, 3)}")
    println("5 * 3 = ${calc.multiply(5, 3)}")
}
